cherrypy/tutorial/docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+version: '3'
+services:
+  tut01:
+    image: ecelis/cherrypy
+    ports:
+      - "8081:8080"
```

Comment:
Could you please dedent sequence? It's nicer to have it right under the mapping key, like:
```yaml
    ports:
    - "8081:8080"
```

Reply:
Done